src/Core/src/Platform/Windows/TransformationExtensions.cs

```diff
@@ -18,9 +18,6 @@ public static void UpdateTransformation(this FrameworkElement frameworkElement,
 			double scaleX = view.Scale * view.ScaleX;
 			double scaleY = view.Scale * view.ScaleY;
 
-			frameworkElement.RenderTransformOrigin = new global::Windows.Foundation.Point(anchorX, anchorY);
-			frameworkElement.RenderTransform = new ScaleTransform { ScaleX = scaleX, ScaleY = scaleY };
-
 			if (rotationX % 360 == 0 && rotationY % 360 == 0 && rotation % 360 == 0 &&
 				translationX == 0 && translationY == 0 && scaleX == 1 && scaleY == 1)
```

Comment:
If these are all default, this `if` should be true.

But are there cases were we need to round? What if someone did some math in their code and `rotationX` is 359.99999999?

Reply:
Maybe someone like @PureWeen or @mattleibow know of an example where we round on other platforms.

I imagine an extension method for `double` like: `IsNearly(0, translationX)` or `IsNearly(1, scaleX)` if we already do this somewhere.